apps/dapp/src/components/Pages/Core/Trade/use-swap-controller.tsx

```diff
@@ -92,9 +91,7 @@ export const useSwapController = () => {
 
   // Handles user input change
   const handleInputChange = async (value: string) => {
-    const bigValue = getBigNumberFromString(value || '0', getTokenInfo(state.inputToken).decimals);
```

Comment:
this prevented 0 from being entered, which makes starting with a decimal hard. updating input value to '0' is fine and won't fetch a quote